External/SDK/Windows/Windows10SDK.bff

```diff
@@ -1,13 +1,39 @@
 // Windows 10 SDK
 //------------------------------------------------------------------------------
-
-// Root Paths
-//------------------------------------------------------------------------------
-.Windows10_SDKBasePath          = '$_CURRENT_BFF_DIR_$/10'
-.Windows10_SDKVersion           = '10.0.17763.0'
+//
+// Detect Windows 10 SDK
+//
+// We search in the following locations, in order of preference:
+//  1) CI Build
+//  2) Vendorized in External (side by side with this bff)
+//  3) Part of a Windows 10 installation (Program Files)
+//
 #if CI_BUILD
     .Windows10_SDKBasePath      = 'C:\Program Files (x86)\Windows Kits/10'
     .Windows10_SDKVersion       = '10.0.17134.0'
+#elseif file_exists( "Windows Kits\10\bin\10.0.19041.0\x64\RC.exe" )
+    //
+    // Use vendorized toolchain
+    //
+    .Windows10_SDKBasePath        = '$_CURRENT_BFF_DIR_$\Windows Kits\10'
+    .Windows10_SDKVersion         = '10.0.19041.0'
```

Comment:
The vendorized toolchain needs to remain on 10.0.17763.0 for now (that's the version being used).
It could likely be updated without much effort, but that has to happen in conjunction with changing the version here